pkg/sequence/group/group.go

```diff
@@ -33,22 +38,190 @@ func init() {
 
 const (
 	SequencePluginName = "group"
+
+	stepKey                 = "step"
+	startSequence     int64 = 1
+	defaultGroupStrep int64 = 100
+
+	initGroupSequenceTableSql = `
+	CREATE TABLE IF NOT EXISTS __arana_group_sequence (
+	    id int AUTO_INCREMENT COMMENT 'primary key',
+		seq_val bigint COMMENT '自增列当前的值',
+		step int COMMENT '自增步长',
+		table_name varchar(255) NOT NULL COMMENT 'arana logic table name',
+	    renew_time datetime NOT NULL COMMENT 'node renew time',
+		PRIMARY KEY (id),
+        UNIQUE KEY(table_name)
+	) ENGINE = InnoDB;
```

Comment:
Format SQL, and translate comments to EN.

Reply:
done